src/components/data/PublicLinks.js

```diff
@@ -186,47 +202,80 @@ function PublicLinks(props) {
                         <Typography variant="caption">{errorMsg}</Typography>
                     </>
                 )}
-                <Toolbar>
-                    <Tooltip title={t("copyLinks")}>
-                        <Button
-                            variant="outlined"
-                            color="primary"
-                            size="small"
-                            className={classes.toolbarButtons}
-                            onClick={copyToClipboard}
-                            startIcon={<FileCopy />}
-                            id={build(baseId, ids.COPY_LINKS_BTN)}
-                        >
-                            <Hidden xsDown>{t("copyLinks")}</Hidden>
-                        </Button>
-                    </Tooltip>
-                    <Tooltip title={t("saveToFile")}>
-                        <Button
-                            variant="outlined"
-                            color="primary"
-                            size="small"
-                            className={classes.toolbarButtons}
-                            onClick={() => setSaveAsDialogOpen(true)}
-                            startIcon={<Save />}
-                            id={build(baseId, ids.SAVE_LINKS_BTN)}
-                        >
-                            <Hidden xsDown>{t("saveToFile")}</Hidden>
-                        </Button>
-                    </Tooltip>
-                    <Tooltip title={t("download")}>
-                        <Button
-                            variant="outlined"
-                            color="primary"
-                            size="small"
-                            className={classes.toolbarButtons}
-                            onClick={() => setDownload(true)}
-                            startIcon={<CloudDownload />}
-                            id={build(baseId, ids.DOWNLOAD_LINKS_BTN)}
-                        >
-                            <Hidden xsDown>{t("download")}</Hidden>
-                        </Button>
-                    </Tooltip>
-                </Toolbar>
+                <TextareaAutosize
+                    rowsMin={paths?.length + 1 || 3}
```

Comment:
Seems like adding logic based on the contents to the `rowsMin` prop in `TextareaAutosize` defeats its purpose?
```suggestion
                    rowsMin={3}
```